src/containers/App/App.js

```diff
@@ -24,7 +25,8 @@ class App extends Component {
 
         <Route path="/login" component={Login} />
         <Route path="/register" component={Register} />
-        {/* <Route path="/items/new" component={NewItem} /> */}
+        <Route path="/items/detail" component={ItemDetail}/>
```

Comment:
make this route dynamic so we can use it to navigate to any item's detail page 
i.e. `<Route path="/items/:id/detail" component={ItemDetail}/>` 
or, `<Route path="/items/:id" component={ItemDetail}/>`
You might need to move it below the items/new route so they dont conflict